kclvm/query/src/override.rs

```diff
@@ -325,7 +326,48 @@ impl<'ctx> MutSelfMutWalker<'ctx> for OverrideTransformer {
             }
         }
 
-        walk_list_mut!(self, walk_stmt, module.body)
+        walk_list_mut!(self, walk_stmt, module.body);
+
+        // If the variable is not found, add a new variable with the override value.
+        if !self.has_override {
```

Comment:
It needs to be determined whether it is a `ast::OverrideAction::CreateOrUpdate` and `has_override` is false